<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Red Cell Compatibility Calculator 3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QGroupBox" name="grp_antigens">
      <property name="title">
       <string>Antigen Selection</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="margin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_N">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Anti-N (MNS System)</string>
         </property>
         <property name="text">
          <string>N</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_M">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Anti-M (MNS System)</string>
         </property>
         <property name="text">
          <string>M</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_C">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Anti-C (Rhesus Antibody)</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_c">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Anti-c (Rhesus Antibody)</string>
         </property>
         <property name="text">
          <string>c</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn_E">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Anti-E (Rhesus Antibody)</string>
         </property>
         <property name="text">
          <string>E</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btn_e">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Anti-e (Rhesus Antibody)</string>
         </property>
         <property name="text">
          <string>e</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="btn_K">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Kell Antigen (Kell System)</string>
         </property>
         <property name="text">
          <string>K</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="btn_k">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Cellano Antigen (Kell System)</string>
         </property>
         <property name="text">
          <string>k</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="btn_Fya">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Anti-Fya (Duffy System)</string>
         </property>
         <property name="text">
          <string>Fya</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btn_S">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Anti-S (MNS System)</string>
         </property>
         <property name="text">
          <string>S</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btn_s">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Anti-s (MNS System)</string>
         </property>
         <property name="text">
          <string>s</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QPushButton" name="btn_Fyb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Anti-Fyb (Duffy System)</string>
         </property>
         <property name="text">
          <string>Fyb</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="btn_Lea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Lewis-A Antibody</string>
         </property>
         <property name="text">
          <string>Lea</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="btn_Leb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Lewis-B Antibody</string>
         </property>
         <property name="text">
          <string>Leb</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="btn_Jka">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Anti-Jka (Kidd System)</string>
         </property>
         <property name="text">
          <string>Jka</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QPushButton" name="btn_Jkb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Anti-Jkb (Kidd System)</string>
         </property>
         <property name="text">
          <string>Jkb</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QPushButton" name="btn_P1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Anti-P1 (P System)</string>
         </property>
         <property name="text">
          <string>P1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,0">
      <item row="0" column="0">
       <widget class="QGroupBox" name="grp_Rh">
        <property name="title">
         <string>Rh Factor</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="rad_pos">
           <property name="text">
            <string>Positive</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rad_neg">
           <property name="text">
            <string>Negative</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="grp_qty">
        <property name="title">
         <string>Quantity</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
         <item row="0" column="1">
          <widget class="QSpinBox" name="spin_qty">
           <property name="statusTip">
            <string>Number of antigen-negative units for the patient</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 14pt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string># of Units:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_reset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="statusTip">
           <string>Restores selections to their original state (F5)</string>
          </property>
          <property name="text">
           <string>  Reset All  </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btn_quit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="statusTip">
           <string>Closes this window and exits the program (Escape)</string>
          </property>
          <property name="text">
           <string>  Exit  </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lab_units">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Units to Antigen-Type:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_output">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="statusTip">
         <string>Automatically updates as selections are made</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font-size: 14pt;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p/&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>345</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRCcompat3">
    <property name="title">
     <string>&amp;Calculator</string>
    </property>
    <addaction name="actionReset_Marked"/>
    <addaction name="actionReset_Antigens"/>
    <addaction name="actionReset_Rh_Factor"/>
    <addaction name="actionReset_Quantity"/>
    <addaction name="separator"/>
    <addaction name="actionConfigure"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionOperation"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt_Framework"/>
   </widget>
   <addaction name="menuRCcompat3"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window...</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>&amp;Configuration...</string>
   </property>
   <property name="statusTip">
    <string>Modify application parameters</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="statusTip">
    <string>Quits the application, no selections are saved</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionOperation">
   <property name="text">
    <string>&amp;Operation / Use...</string>
   </property>
   <property name="statusTip">
    <string>Displays help information</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About RCcompat3...</string>
   </property>
   <property name="statusTip">
    <string>Displays information about the application</string>
   </property>
  </action>
  <action name="actionAbout_Qt_Framework">
   <property name="text">
    <string>About the &amp;Qt Framework...</string>
   </property>
   <property name="statusTip">
    <string>Displays information about Qt</string>
   </property>
  </action>
  <action name="actionReset_Marked">
   <property name="text">
    <string>&amp;Reset All</string>
   </property>
   <property name="statusTip">
    <string>Clears all selections and sets Quantity to 1</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionReset_Antigens">
   <property name="text">
    <string>Reset &amp;Antigens</string>
   </property>
   <property name="statusTip">
    <string>Clears only the Antigen Selections</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionReset_Rh_Factor">
   <property name="text">
    <string>Reset Rh &amp;Factor</string>
   </property>
   <property name="statusTip">
    <string>Clears only the Rh Factor selection</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionReset_Quantity">
   <property name="text">
    <string>Reset &amp;Quantity</string>
   </property>
   <property name="statusTip">
    <string>Sets Quantity back to 1</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btn_C</tabstop>
  <tabstop>btn_c</tabstop>
  <tabstop>btn_E</tabstop>
  <tabstop>btn_e</tabstop>
  <tabstop>btn_K</tabstop>
  <tabstop>btn_k</tabstop>
  <tabstop>btn_Fya</tabstop>
  <tabstop>btn_Fyb</tabstop>
  <tabstop>btn_M</tabstop>
  <tabstop>btn_N</tabstop>
  <tabstop>btn_S</tabstop>
  <tabstop>btn_s</tabstop>
  <tabstop>btn_Lea</tabstop>
  <tabstop>btn_Leb</tabstop>
  <tabstop>btn_Jka</tabstop>
  <tabstop>btn_Jkb</tabstop>
  <tabstop>btn_P1</tabstop>
  <tabstop>rad_pos</tabstop>
  <tabstop>rad_neg</tabstop>
  <tabstop>spin_qty</tabstop>
  <tabstop>btn_reset</tabstop>
  <tabstop>btn_quit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
